build/rust/config.gni

```diff
@@ -47,7 +47,8 @@ if (use_lld) {
 
 # https://bugzilla.mozilla.org/show_bug.cgi?id=1188030#c14
 # Supposedly this bug was fixed, but it's still happening for us.
-too_many_personality_profiles_workaround = is_apple && use_lld && !is_component_build
```

Comment:
why is this being changed in this PR?

Reply:
linter has gone crazy :(

Reply:
Because this PR is modifying a lot of places, I'd like to propose backing out this lint change (and any any others not absolutely needed). If the file was being modified already, then lint change makes sense. But general lint cleanups can happen in a different PR

Reply:
reverted